### PR TITLE
Add "logo" to accessible name of images where relevant

### LINKED ISSUE #1806: Include "logo" in a logo image's accessible name [qt1vmo]
For test case examples that use a logo for an image, the accessible name for the image should include that it is a logo. 

In this [Passed Example 1](https://act-rules.github.io/rules/qt1vmo#passed-example-1) for Image accessible name is descriptive; it should be alt="W3C logo".  

`W3C`

### PR DESCRIPTION
**Pull request by Jym77**
Add "logo" to the name of images that are logos, where it make sense.
I feel it doesn't make sense in cases like
```html
<a href="https://act-rules.github.io/"><img src="test-assets/shared/act-logo.png" alt="ACT rules"/></a>
```
where the image is functional rather than informative, so leaving those as they are.

Closes issue(s):

- closes #1806

Need for Call for Review:
This will require a 1 week Call for Review (improving test cases)

---

### **When creating PR:**

- [X] Make sure you're requesting to **pull a branch** (right side) to the `develop` branch (left side).
- [X] Make sure you do not remove the "How to Review and Approve" section in your pull request description

### **After creating PR:**

- [X] Add yourself (and co-authors) as "Assignees" for PR.
- [X] Add label to indicate if it's a `Rule`, `Definition` or `Chore`.
- [X] [Link the PR to any issue it solves](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue). This will be done automatically by [referencing the issue](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) at the top of this comment in the indicated place.
- [X] Optionally request feedback from anyone in particular by assigning them as "Reviewers".

### **When merging a PR:**

- [ ] Close any issue that the PR resolves. This will happen automatically upon merging if the PR was correctly linked to the issue, e.g. by referencing the issue at the top of this comment.

## How to Review And Approve

- Go to the “Files changed” tab
- Here you will have the option to leave comments on different lines.
- Once the review is completed, find the “Review changes” button in the top right, select “Approve” (if you are really confident in the rule) or "Request changes" and click “Submit review”.
- Make sure to also review the proposed Call for Review period. In case of disagreement, the longer period wins.

### CODE REVIEW

**Review by carlosapaduarte — APPROVED** [review]

**Review by HelenBurge — APPROVED** [review]

**Review by kengdoj — APPROVED** [review]

**Comment by Jym77** [comment]
Call for review ends on September 15.

File: _rules/scrollable-element-keyboard-accessible-0ssw9k.md [hunk 1]
```diff
@@ -146,13 +146,13 @@ This [horizontally scrollable][scrollable] `section` element is not included in
...
-	<img src="/test-assets/shared/w3c-logo.png" alt="W3C" />
+	<img src="/test-assets/shared/w3c-logo.png" alt="W3C logo" />
+	<img src="/test-assets/shared/eu-logo.svg" alt="EU logo" />
+	<img src="/test-assets/shared/w3c-logo.png" alt="W3C logo" />
+	<img src="/test-assets/shared/eu-logo.svg" alt="EU logo" />
+	<img src="/test-assets/shared/w3c-logo.png" alt="W3C logo" />
+	<img src="/test-assets/shared/eu-logo.svg" alt="EU logo" />
+	<img src="/test-assets/shared/w3c-logo.png" alt="W3C logo" />
 </section>
 ```
```
Comment (giacomo-petri) [inline _rules/scrollable-element-keyboard-accessible-0ssw9k.md]: From my experience these kind of components are visually managed including logos for a stronger visual impact, but their real purpose is to just list a list of brands.

I don't think we should change these examples including the "logo" word into the image alternative, that is not adding relevant information for the user.

Comment (Jym77) [reply _rules/scrollable-element-keyboard-accessible-0ssw9k.md]: Ah, yes, especially in a "our sponsors" list 😅

**Review by giacomo-petri — APPROVED** [review]

**Comment by Jym77** [comment]
Call for reviews has ended, merging.

**Pull request merged**